### ISSUE TITLE
Unable to load image

### ISSUE THREAD

**Issue by karan1213** [chat-sdk/chat-sdk-android#396]
```
Fatal Exception: io.reactivex.exceptions.UndeliverableException
java.lang.Throwable: Unable to load image
Caused by java.lang.Throwable: Unable to load image
       at co.chatsdk.ui.utils.ImageBuilder$1.onFailureImpl(ImageBuilder.java:47)
       at com.facebook.datasource.BaseDataSubscriber.onFailure(BaseDataSubscriber.java:59)
       at com.facebook.datasource.AbstractDataSource$1.run(AbstractDataSource.java:174)
       at com.facebook.common.executors.CallerThreadExecutor.execute(CallerThreadExecutor.java:49)
       at com.facebook.datasource.AbstractDataSource.notifyDataSubscriber(AbstractDataSource.java:169)
       at com.facebook.datasource.AbstractDataSource.notifyDataSubscribers(AbstractDataSource.java:160)
       at com.facebook.datasource.AbstractDataSource.setFailure(AbstractDataSource.java:237)
       at com.facebook.imagepipeline.datasource.AbstractProducerToDataSourceAdapter.onFailureImpl(AbstractProducerToDataSourceAdapter.java:84)
       at com.facebook.imagepipeline.datasource.AbstractProducerToDataSourceAdapter.access$000(AbstractProducerToDataSourceAdapter.java:28)
       at com.facebook.imagepipeline.datasource.AbstractProducerToDataSourceAdapter$1.onFailureImpl(AbstractProducerToDataSourceAdapter.java:56)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.MultiplexProducer$Multiplexer.onFailure(MultiplexProducer.java:412)
       at com.facebook.imagepipeline.producers.MultiplexProducer$Multiplexer$ForwardingConsumer.onFailureImpl(MultiplexProducer.java:506)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DelegatingConsumer.onFailureImpl(DelegatingConsumer.java:28)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DecodeProducer$ProgressiveDecoder.handleError(DecodeProducer.java:361)
       at com.facebook.imagepipeline.producers.DecodeProducer$ProgressiveDecoder.onFailureImpl(DecodeProducer.java:188)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DelegatingConsumer.onFailureImpl(DelegatingConsumer.java:28)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DelegatingConsumer.onFailureImpl(DelegatingConsumer.java:28)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.MultiplexProducer$Multiplexer.onFailure(MultiplexProducer.java:412)
       at com.facebook.imagepipeline.producers.MultiplexProducer$Multiplexer$ForwardingConsumer.onFailureImpl(MultiplexProducer.java:506)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DelegatingConsumer.onFailureImpl(DelegatingConsumer.java:28)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.DelegatingConsumer.onFailureImpl(DelegatingConsumer.java:28)
       at com.facebook.imagepipeline.producers.BaseConsumer.onFailure(BaseConsumer.java:112)
       at com.facebook.imagepipeline.producers.NetworkFetchProducer.onFailure(NetworkFetchProducer.java:189)
       at com.facebook.imagepipeline.producers.NetworkFetchProducer.access$100(NetworkFetchProducer.java:35)
       at com.facebook.imagepipeline.producers.NetworkFetchProducer$1.onFailure(NetworkFetchProducer.java:75)
       at com.facebook.imagepipeline.producers.HttpUrlConnectionNetworkFetcher.fetchSync(HttpUrlConnectionNetworkFetcher.java:86)
       at com.facebook.imagepipeline.producers.HttpUrlConnectionNetworkFetcher$1.run(HttpUrlConnectionNetworkFetcher.java:60)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:422)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
```

I am using ChatSDK 4.0.26
Its getting crashed when loading image on ChatActivity.

In Firebase I saw that the entry for pictureURL is "file:///data/user/0/com.<package_name>/cache/cropped-1038003443.jpg"

Is this correct way of saving picture url?

**Comment by bensmiley** [comment]
@karan1213 Could you update to version `4.2.9` and see if the project persists? We've made some fies around image handling in the last few versions.

**Comment by bensmiley** [comment]
Fixed in 4.6.0.
